File: TYPESCRIPT PROGRAMMING/Test72.ts
// 72. PRIME NUMBERS FROM 0 TO N
const prime = (num: number): boolean => {
    let flag: boolean = true;
    for(let i = 2; i <= Math.floor(Math.sqrt(num)); i++) {
        if(num % i === 0) flag = false;
    }
    return flag;
}
const primeSeries = (range: number): void => {
    for(let i = 2; i <= range; i++) {
        if(prime(i)) {
            process.stdout.write(`${i} `);
        }
    }
    console.log();
}
primeSeries(100);
primeSeries(200);

File: JAVA PROGRAMMING/Test72.java
// 72. PRIME NUMBERS FROM 0 TO N
import java.util.Scanner;
public class Test {
    public static boolean isPrime(int num) {
        boolean flag = true;
        for(int i = 2; i <= Math.sqrt(num); i++) {
            if(num % i == 0) {
                flag = false;
            }
        }
        return flag;
    }
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        for(int i = 2; i <= range; i++) {
            if(isPrime(i)) {
                System.out.print(i+" ");
            }
        }
        in.close();
    }
}